src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,21 @@
 package mate.academy.service;
 
+import java.util.Objects;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+
+        return Objects.requireNonNull(UserService.findByEmail(email)).getEmail().equals(email)
+                && Objects.requireNonNull(UserService.findByEmail(email)).getPassword()
+                .equals(password);
```

Comment:
You don`t need use class Objects here, at first you can get user form UserService.findByEmail(email) anf after this compare his password